CHANGES.rst

```diff
@@ -430,6 +430,12 @@ astropy.modeling
 - Added ``is_separable`` function to modeling to test the
   separability of a model. [#6746]
 
+- Added ``Model.separable`` property. It returns a boolean value or
+  ``None`` if not set. [#6746]
+
+astropy.nddata
```

Comment:
This doesn't look right.

Reply:
Not sure what happened here. However this is probably OK but the entire entry for nddata (just below) looks wrong.